website_version_ce/controllers/main.py

```diff
@@ -0,0 +1,221 @@
+# -*- coding: utf-8 -*-#
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+import datetime
+from openerp import http
+from openerp.http import request
+from openerp.addons.website.controllers.main import Website
+from openerp.tools.misc import DEFAULT_SERVER_DATETIME_FORMAT
+
+GOOGLE_ANALYTICS_CONFIGURED = 2
+GOOGLE_ANALYTICS_PARTIALLY_CONFIGURED = 1
+GOOGLE_ANALYTICS_NOT_CONFIGURED = 0
+
+
+class VersioningController(Website):
+
+    @http.route('/website_version_ce/change_version', type='json', auth="user",
```

Comment:
Well, _website_version_oca_ is also not the best name, as is not a version developed by OCA people, but only the Odoo's one that is open source. So I think the suffix `_ce` is appropiated.